pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -183,6 +183,16 @@ func validateExternalAdmissionHook(hook *admissionregistration.ExternalAdmission
 	if hook.FailurePolicy != nil && *hook.FailurePolicy != admissionregistration.Ignore {
 		allErrors = append(allErrors, field.NotSupported(fldPath.Child("failurePolicy"), *hook.FailurePolicy, []string{string(admissionregistration.Ignore)}))
 	}
+	if len(hook.URLPath) != 0 {
+		steps := strings.Split(hook.URLPath, "/")
```

Comment:
`/foo/bar/` will validate `["","foo","bar",""]` and fail the first and last steps, right?

Reply:
if non-empty, must also start with a `/`

Reply:
> if non-empty, must also start with a /

Ah, I was going the other way, no leading or trailing.  If you like though.  You want to force trailing or deny it?  I don't want to allow both.